proto/zitadel/policy/v2alpha/policy_service.proto

```diff
@@ -0,0 +1,316 @@
+                key: "200"
+                value: {
+                    description: "OK";
+                }
+            };
+        };
+    }
+    rpc GetDomainSettings (GetDomainSettingsRequest) returns (GetDomainSettingsResponse) {
+        option (google.api.http) = {
+            get: "/v2alpha/policies/domain"
+        };
+
+        option (zitadel.v1.auth_option) = {
+            permission: "policy.read"
+        };
+
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "Get the current active domain settings";
+            description: "Return the current active domain settings for the requested context"
+            responses: {
+                key: "200"
+                value: {
+                    description: "OK";
+                }
+            };
+        };
+    }
+    rpc GetLegalSettings (GetLegalSettingsRequest) returns (GetLegalSettingsResponse) {
+        option (google.api.http) = {
+            get: "/v2alpha/policies/legal"
+        };
+
+        option (zitadel.v1.auth_option) = {
+            permission: "policy.read"
+        };
+
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "Get the current active legal settings";
+            description: "Return the current active legal settings for the requested context"
+            responses: {
+                key: "200"
+                value: {
+                    description: "OK";
+                }
+            };
+        };
+    }
+    rpc GetLockoutPolicy (GetLockoutPolicyRequest) returns (GetLockoutPolicyResponse) {
+        option (google.api.http) = {
+            get: "/v2alpha/policies/lockout"
+        };
+
+        option (zitadel.v1.auth_option) = {
+            permission: "policy.read"
+        };
+
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "Get the current active lockout policy";
+            description: "Return the current active lockout policy for the requested context"
+            responses: {
+                key: "200"
+                value: {
+                    description: "OK";
+                }
+            };
+        };
+    }
+    rpc GetActiveIdentityProviders (GetActiveIdentityProvidersRequest) returns (GetActiveIdentityProvidersResponse) {
+        option (google.api.http) = {
+            get: "/v2alpha/policies/idps"
+        };
+
+        option (zitadel.v1.auth_option) = {
+            permission: "policy.read"
+        };
+
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "Get the current active identity providers";
+            description: "Return the current active identity providers for the requested context"
+            responses: {
+                key: "200"
+                value: {
+                    description: "OK";
+                }
+            };
+        };
+    }
```

Comment:
i assume we only return active idps here and will have an additional request to request idps generally later on

Reply:
exactly, the idea is just to provide all the endpoints to get necessary information for login and register